packages/mds-utils/utils.ts

```diff
@@ -42,6 +49,26 @@ const RADIUS = 30.48 // 100 feet, in meters
 const NUMBER_OF_EDGES = 32 // Number of edges to add, geojson doesn't support real circles
 const UUID_REGEX = /[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}/
 
+/* Is `as` a subset of `bs`? */
+const isSubset = <T extends Array<string>, U extends Readonly<Array<string>>>(as: T, bs: U) => {
+  return !as.some(a => !bs.includes(a))
+}
+
+const isMicroMobilityEvent = (
+  device: Pick<Device, 'modality'>,
```

Comment:
Does `{ modality }: Device` destructuring not work in this context?
If not, `{ modality }: Pick<Device, 'modality'>` would.